docs/update-database.rst

```diff
@@ -0,0 +1,98 @@
+Update a database
+=================
+
+A feature that is especially useful for databases growing over time,
+is the possibility to update a :class:`audformat.Database`.
+For instance, consider the following database that contains
+age labels for a hundred files.
+
+.. jupyter-execute::
+
+    import audformat
+    import audformat.testing
+
+
+    db = audformat.testing.create_db(minimal=True)
+    db.schemes['age'] = audformat.Scheme(
+        audformat.define.DataType.INTEGER,
+        minimum=20,
+        maximum=50,
+    )
+    audformat.testing.add_table(
+        db,
+        table_id='table',
+        index_type=audformat.define.IndexType.FILEWISE,
+        columns='age',
+        num_files=100,
+    )
+    db
+
+.. jupyter-execute::
+
+    db['table'].df
+
+Now assume we record some more files that we want add to our original database.
+To not confuse new and old data, we start from a fresh database containing
+only new files, get the annotations and update the original database.
```

Comment:
This sentence was not immediately clear to me as it wasn't obvious on wish database is meant by "fresh database".
I now get it, but maybe we can improve here:

```rst
The new files are stored together with annotations in a second database,
that is then added to the original database.
```

Reply:
Updated accordingly.